crates/red_knot_python_semantic/src/types.rs

```diff
@@ -233,31 +233,39 @@ fn declarations_ty<'db>(
 /// Unique ID for a type.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub enum Type<'db> {
-    /// the dynamic type: a statically-unknown set of values
+    /// The dynamic type: a statically-unknown set of values
     Any,
-    /// the empty set of values
+    /// The empty set of values
     Never,
-    /// unknown type (either no annotation, or some kind of type error)
-    /// equivalent to Any, or possibly to object in strict mode
+    /// Unknown type (either no annotation, or some kind of type error)
+    /// Equivalent to Any, or possibly to object in strict mode
```

Comment:
If you want the second line to be a second sentence, you need a period at the end of the first line ;)

```suggestion
    /// Unknown type (either no annotation, or some kind of type error).
    /// Equivalent to Any, or possibly to object in strict mode
```

Reply:
Ah sorry I just noticed that some comments started with a capital and some didn't, maybe didn't entirely pay attention